src/actionparameters.ts

```diff
@@ -131,4 +134,9 @@ export class ActionParameters {
     public get multiContainerConfigFile() {
         return this._multiContainerConfigFile;
     }
+
+    private _sanitizeInputs() {
+        this._appName = this._appName.toLowerCase();
+        this._slotName = this._slotName.toLowerCase();

```

Comment:
Instead of changing appName and slotName to lower case. can we just check it at the time of validations https://github.com/Azure/webapps-deploy/blob/master/src/ActionInputValidator/Validations.ts#L33

Reply:
Done